model/resource/os.yaml

```
@@ -63,3 +63,7 @@ groups:
           The version string of the operating system as defined in
           [Version Attributes](/docs/resource/README.md#version-attributes).
         examples: ['14.2.1', '18.04.1']
+      - id: build
```

Maybe this should be named

```suggestion
      - id: build_id
```